projects/vdk-plugins/vdk-meta-jobs/src/vdk/plugin/meta_jobs/plugin_entry.py

```diff
@@ -13,9 +13,19 @@
 
 
 class MetaJobsPlugin:
```

Comment:
nit: the `plugin_entry.py` contains MetaJobsPlugin class, and a static method; why not using `<feature>-plugin.py` naming convention that is common to probably most of the plugins we do?

Reply:
you are right, it seems like a good idea to change it to dags_plugin.py

Reply:
Done.